src/components/App.tsx

```diff
@@ -159,7 +159,24 @@ export const App = () => {
     const maxDate = endDate || new Date(Date.now());
     guaranteeGlobal(globalMinDate, Number(minDate)/1000);
     guaranteeGlobal(globalMaxDate, Number(maxDate)/1000);
-  }, [endDate, startDate, weatherStations]);
+    if (weatherStation && (!startDate || !endDate)) {
+      setStatus({
+        status: "error",
+        message: "Select a date range",
+        icon: <WarningIcon/>
+      });
+    }
+    // if the start date of the weather station is after the user's requested end date, then the station is inactive
+    const stationMinDate = new Date(weatherStation?.mindate || globalMinDate);
+    const stationNotActiveYet = stationMinDate > maxDate;
```

Comment:
Note on date range checks:
We currently only check if the station's start date is after the user's end date. We don't check the opposite (station's end date before user's start date) because our source data's end dates are not up-to-date (see note in PT-187533021) Implementing the opposite check could therefore result in false negatives for active stations.